gen/azure/cloud-config.yaml

```diff
@@ -74,7 +76,7 @@ bootcmd:
 disk_setup:
   ephemeral0:
     table_type: gpt
-    layout: [ 50, 50 ]
+    layout: [ 25, 50, 25 ]
```

Comment:
So internally we are currently planning on restricting vm sizes to only vms that have at least a 100GB ephemeral disk. Does that seem reasonable to you? I could update this template and then set layout to 44,44,2.